atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/KvsSweepQueueTest.java

```diff
@@ -195,6 +188,169 @@ public void sweepDeletesTombstonesWhenWriteHasHigherTimestamp() {
         assertReadAtTimestampReturnsValue(TABLE_CONSERVATIVE, TS2 + 1, TS2);
     }
 
+    @Test
+    public void sweepProgressesAndSkipsEmptyFinePartitions() {
```

Comment:
This test could use some comments to explain the state of the world at each stage. (same for various other tests in this class)